ecommerce_integrations/unicommerce/pick_list.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+import frappe
+from frappe import _
+
+
+def validate(self, method=None):
```

Comment:
This validation should only apply if Pick List is generated for unicommerce orders? 


Similar to my comments on Sales Invoice, make sure that other use cases don't break while validating for unicommerce.